khal/parse_datetime.py

```diff
@@ -413,12 +413,20 @@ def guessrangefstr(daterange, locale,
     )
 
 
-def rrulefstr(repeat, until, locale):
+def rrulefstr(repeat, until, every, locale):
     if repeat in ["daily", "weekly", "monthly", "yearly"]:
         rrule_settings = {'freq': repeat}
         if until:
             until_dt, is_date = guessdatetimefstr(until.split(' '), locale)
             rrule_settings['until'] = until_dt
+        if every:
+            try:
+                int(every)
+            except ValueError:
```

Comment:
It would be clearer here (and more efficient, not that it really matters) to use `if every.isdigit()` here, as you're not using the cast.

Reply:
Yes, I and I later discovered that prompt had a type feature to constraint the input which is a better way of handling it, and later learned that it also have to be positive to I have to change this anyhow.

Reply:
fwiw `.isdigit()` would return false for negatives :)

Reply:
isdigit() ftw :-)

Reply:
Well, turns out retrieves values are of type vInt so cannot use isdigit() on that, and I need to check for int(every) > 0.  Just a heads up if you wonder why I didn't actually fixed it.